nixos/modules/services/misc/matrix-synapse.nix

```diff
@@ -118,6 +121,14 @@ in {
           Overridable attribute of the matrix synapse server package to use.
         '';
       };
+      plugins = mkOption {
+        type = types.listOf types.package;
+        default = with pkgs.matrix-synapse-plugins; [ matrix-synapse-ldap3 matrix-synapse-pam ];
+        defaultText = "with pkgs.matrix-synapse-plugins [ matrix-synapse-ldap3 matrix-synapse-pam ]";
```

Comment:
Maybe those plugins shouldn't be enabled by default, in particular the newly introduced one.